### PR TITLE
fix: show app latest release 1148

### LINKED ISSUE #1148: Show app’s “last updated”
In the list of Electron apps, there is a `Added (x <time>) ago` bit of info in the long list view.

Many of these apps were added 2 years ago, which can give the appearance of staleness.  

Instead, I’d love to see this same info show when each app was last **updated**.

### PR DESCRIPTION
**Pull request by nhorilopchan**
Resolve #1148 Show "Latest Release" date on listing

### CODE REVIEW

File: scripts/update-app-download-links.js [hunk 1]
```diff
@@ -15,6 +14,7 @@ module.exports = function updateAppDownloadLinks () {
   if (!matches.length) {
     // No downloads found for the detected OS, probably android/ios
     // Hide all links and let them click view all files if wanted
+    links[0].parentNode.style.display = 'none'
```
Comment (zeke) [inline scripts/update-app-download-links.js]: Looks like this is a leftover change from a different PR.

Comment (nhorilopchan) [reply scripts/update-app-download-links.js]: yes sorry about that

Comment (nhorilopchan) [reply scripts/update-app-download-links.js]: There is another commit that has the fix for this issue though.

**Comment by this8is8mine8** [comment]
Try it

**Comment by ckerr** [comment]
This PR is quite old and has built up some conflicts over time.

@nhorilopchan could you please resolve the conflicts?

**Comment by ckerr** [comment]
I'm closing this PR for now due to lack of response.

@nhorilopchan please feel free to open a new PR for this issue -- I'd be happy to see an up-to-date PR get merged.

**Pull request closed without merging**